platform-includes/performance/create-transaction-bound-to-scope/java.mdx

```diff
@@ -2,6 +2,8 @@
 
 Our SDK can bind a transaction to the scope making it accessible to every method running within this scope by calling `Sentry#startTransaction` method with `bindToScope` parameter to `true`.
 
+Our auto instrumentation, for example those instrumenting database requests or HTTP calls, require a transaction to be bound to the current scope in order to attach a new child span.
```

Comment:
IIRC in order to attach messages/exceptions/breadcrumbs we also need to have the transaction on the scope, correct? In that case, should we mention that also?

Reply:
Most confusion (customers opening issues / asking questions) seems to come from missing spans at the moment. If this comes up again we can add it, but it probably shouldn't be in Performance docs but somewhere else.